edx_proctoring/templates/proctored_exam/expired.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      The due date for this exam has passed
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since the due date for this exam has already passed, you are no longer be able to take this examination.
+    {% endblocktrans %}
+  </p>
+  <hr>
+  <p>
+    {% blocktrans %}
+      View your credit eligibility status on your <a href="{{progress_page_url}}">Progress</a> page.
+    {% endblocktrans %}
+  </p>
+</div>
+{% include 'proctored_exam/footer.html' %}
```

Comment:
@mhoeber can you do a review on this little bit of content? Use case here is when a user clicks on a proctored exam when it is past the exam due date.